Resources/Prototypes/_Impstation/CosmicCult/mind_roles.yml

```diff
@@ -0,0 +1,19 @@
+- type: entity
+  parent: BaseMindRoleAntag
+  id: MindRoleCosmicCult
+  name: Cosmic Cult Role
+#  description: mind-role-cosmiccult-description
+  components:
+  - type: MindRole
+    antagPrototype: CosmicAntagCultist
+  - type: CosmicCultRole
+
+- type: entity
+  parent: MindRoleCosmicCult
+  id: MindRoleCosmicCultLead
+  name: Cosmic Cult Leader Role
+#  description: mind-role-cosmiccult-description
```

Comment:
Remove the commented out descriptions, or add them back in. Don't leave them commented out